src/containers/Home/Jobs/Pagination/index.tsx

```diff
@@ -0,0 +1,22 @@
+export default function Pagination(props: { jobsPerPage: number, totalJobs: number, paginate: any, currentPage:number }) {
+    const { jobsPerPage, totalJobs, paginate,currentPage } = props
+    const pageNumber = []
+    for (let i = 1; i <= Math.ceil(totalJobs / jobsPerPage); i++) {
+        pageNumber.push(i)
+    }
```

Comment:
should use useEffect to handle this logic

Reply:
i've changed